content/hardware/03.nano/boards/nano-esp32/tutorials/spiff/spiff.md

```diff
@@ -0,0 +1,80 @@
+---
+title: SPIFFS Partition on Nano ESP32
+description: Get started with the SPIFFS partition scheme on the Nano ESP32.
+author: Hannes Siebeneicher
+hardware:
+  - hardware/03.nano/boards/nano-esp32
+tags: [ESP32, SPIFFS, IDE]
+---
+
+The SPIFFS (Serial Peripheral Interface Flash File System) is a file system designed to be used with NOR flash memory devices and microcontrollers that support the Serial Peripheral Interface (SPI) protocol such as the ESP32 found on the [Nano ESP32](https://store.arduino.cc/products/nano-esp32?queryID=undefined).
+
+***Before proceeding with the SPIFFS file system setup on Nano ESP, please be aware that the mounting instructions mentioned in a comment on the code from the packaged built-in example are tailored for specific older IDE versions such as IDE 1.8. These instructions might not be relevant to IDE 2.0, as it does not support the utilization of custom tools, which are unnecessary in this case.***
+
+## Software & Hardware Needed
+
+- [Arduino Nano ESP32](https://store.arduino.cc/nano-esp32)
+- [Arduino IDE](https://www.arduino.cc/en/software)
+- [Arduino ESP32 Core](https://github.com/arduino/arduino-esp32)
+
+## Set Up SPIFFS File System
+
```

Comment:
@Hannes7eicher can you add a quick summary of what the example does here, like 
- "To test out SPIFFS, we are going to use the `SPIFFS_Test` example. This example does X, then Y, then Z etc. 

Just a quick summary so that the reader knows what to expect.

Reply:
I added the example below together with a section explaining the code.

Reply:
Looks great, thank you @Hannes7eicher